content/for-cisos/data-protection/rhacm-data-storage.md

```diff
@@ -0,0 +1,11 @@
+
+# **Data storage**
```

Comment:
No need to make heading bold, avoid the **TITLE** markup

Reply:
updated